adapters/test-adapter/test-adapter.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * test-adapter.js - Test Adapter.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.*
+ */
+
+'use strict';
+
+var Adapter = require('../adapter');
+var Device = require('../device');
+var Property = require('../property');
+
+class TestProperty extends Property {
```

Comment:
Would it be accurate to refer to the TestAdapter as a MockAdapter?

MockProperty
MockDevice
MockAdapter